q2_moshpit/plugin_setup.py

```diff
@@ -287,6 +287,29 @@
     citations=[citations["wood2019"], citations["lu2017"]],
 )
 
+plugin.methods.register_function(
+    function=q2_moshpit.kraken2.database.inspect_kraken_db,
```

Comment:
Could we maybe change it to `inspect_kraken2_db`? Just to be explicit about which Kraken version we're using.